integration_testing/features/super-admin/super-admin-import-individual-users.feature

```diff
@@ -0,0 +1,152 @@
+Feature: Import individual users
```

Comment:
Let's just change the filename to reflect that it's a part of the setup wizard procedure:
`super-admin-setup-wizard-import-individual-users.feature`